src/java.desktop/share/classes/javax/swing/text/InternationalFormatter.java

```diff
@@ -528,9 +528,7 @@ void updateMask() {
 
                         updateMask(iterator);
                     }
-                    catch (ParseException pe) {}
-                    catch (IllegalArgumentException iae) {}
-                    catch (NullPointerException npe) {}
+                    catch (ParseException | NullPointerException | IllegalArgumentException e) {}
```

Comment:
Maybe move `catch` to the line with the closing brace? The try block above follows the regular Java code style.